Components/Footer.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
```

Comment:
Good job making the changes requested by the previous reviewer @anderCM, But there are some commented linter rules. It would be wise to remove all. Remember linters are here to help you write clean code.